src/Compilers/Shared/BuildClient.cs

```diff
@@ -51,23 +54,23 @@ internal RunCompilationResult RunCompilation(IEnumerable<string> originalArgumen
 
             var args = originalArguments.Select(arg => arg.Trim()).ToArray();
 
-            bool hasShared;
-            string keepAlive;
-            string errorMessage;
-            string sessionKey;
-            List<string> parsedArgs;
             if (!CommandLineParser.TryParseClientArgs(
                     args,
-                    out parsedArgs,
-                    out hasShared,
-                    out keepAlive,
-                    out sessionKey,
-                    out errorMessage))
+                    out List<string> parsedArgs,
+                    out bool hasShared,
+                    out string keepAlive,
+                    out string sessionKey,
+                    out string errorMessage))
```

Comment:
>out string errorMessage)) [](start = 20, length = 25)

IMHO, changes like that do not make code more readable. I would say quite the opposite.